entities/interview.py

```diff
@@ -0,0 +1,39 @@
+import openai
+import uuid
+from .question import Question
+
+class Interview(object):
```

Comment:
This class looks like a wrapper of Question if i understand correctly - i'm not sure if we need this but the methods defined here help me understand the logic so let's keep this

please see my comments below which hopefully explains how you can use Question class

Reply:
Yes, I am also thinking about the proper way to define the base entity while writing this yesterday, each instance of the entity actually is an interview conversation,  so I was thinking that a single question could be a lower level concept,  one interview conversation could involve multiple questions, so I write this in this way

but from the chat gpt api perspective,  there is a concept of "messages", or call it conversation history, this will help gpt side to build the context and talk like a human, this will need higher level information of a single Question.

Additionally,  Question is our own biz type,  and messages/history is gpt specific concept, so we'd better to distinguish them.

Based all above thinking, in anyways I think we need a higher level concept beyond Question, either conversation or interview.

https://platform.openai.com/docs/api-reference/chat/create
for your reference